ProjectSettings/TagManager.asset

```diff
@@ -5,6 +5,7 @@ TagManager:
   serializedVersion: 2
   tags:
   - Enemy
+  - Bullet
```

Comment:
???

Reply:
Enemy hitbox now detects bullets by tag, not name